tests/integration/testdata/buildcmd/terraform/zip_based_lambda_functions_local_backend_windows/src/list_books/requirements.txt

```diff
@@ -0,0 +1,2 @@
+boto3
+ptvsd
```

Comment:
we can remove ptvsd from requirements as well.

Reply:
will remove it through another PR